src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -46,6 +51,10 @@ export interface IEditorPageProps extends RouteComponentProps, React.Props<Edito
     applicationActions: IApplicationActions;
 }
 
+interface IExportPageSettings extends IAssetPreviewSettings {
```

Comment:
If this interface is going to exist as a prop on the page it needs to be exported and it should be moved up into `applicationState` with our application level interfaces.